springql-core/src/api/low_level_rs/spring_config.rs

```diff
@@ -75,9 +75,9 @@ pub fn spring_config_default() -> SpringConfig {
 ///
 /// # Failures
 ///
-/// - [SpringError::InvalidConfig](crate::error::SpringError::InvalidConfig) when:
+/// - [SpringError::InvalidConfig](crate::api::error::SpringError::InvalidConfig) when:
 ///   - `overwrite_config_toml` includes invalid key and/or value.
-/// - [SpringError::InvalidFormat](crate::error::SpringError::InvalidFormat) when:
+/// - [SpringError::InvalidFormat](crate::api::error::SpringError::InvalidFormat) when:
 ///   - `overwrite_config_toml` is not valid as TOML.
 pub fn spring_config_toml(overwrite_config_toml: &str) -> Result<SpringConfig> {
```

Comment:
I guess we can remove this function (and write wrapper codes calling `SpringConfig::new(overwrite_config_toml)` in SpringQL-client-c/src/lib.rs ).